plugins/funind/glob_termops.mli

```diff
@@ -122,3 +122,10 @@ val zeta_normalize : Glob_term.glob_constr -> Glob_term.glob_constr
 
 
 val expand_as : glob_constr -> glob_constr
+
+
+(* [resolve_and_replace_implicits ?expected_type env sigma rt] solves implicits of [rt] w.r.t. [env] and [sigma] and then replace them by their solution 
+ *)
+val resolve_and_replace_implicits :
```

Comment:
Oups, I'm tired. The placeholders for implicit arguments are there but it is their value which is requested. But then, this comforts me that it would be simpler to stay at the level of `constr` rather than going back to `glob_constr`.